index.html

```diff
@@ -0,0 +1,141 @@
+                            <li class="disable"><a href="#">Pricing</a></li>
+                            <li class="disable"><a href="#">Contact us</a></li>
+                            <li class="campain"><a href="">CC Campain</a></li>
+                          </ul>
+                        </nav>
+                    </div> 
+                    <div>
+                      
+                    </div>
+                    <div class="main">
+                      <h2>"Hello Sharing World"</h2>
+                      <h1>WELCOM TO THE BEST GRILL</h1>
+                       <p>Get All Types of Sea Food</p>
+
+                       <div class="sign">
+                         <h3>2022.04.15(MON) ~ (FRI)</h3>
+                         <h5>@information share Saraza</h5>
+                       </div>
+                      </div>
+             </header>
+             <section>
+                 <aside class="dishes">
+                   <div class="title">
+                    <h4>SIGNATURE DISHES</h4>
+                    <div class="bar"></div>
+                   </div>
+                    
+                    <div class="cards-container">
+                      <div class="cards">
+                          <img src="./components/service1.jpg" alt="service1 image">
+                          <div>
+                               <h4>Grill</h4>
+                               <p>
+                                   Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                          </div>
+                      </div>
+                      <div class="cards">
+                          <img src="./components/service2.jpg"  alt="service2 image">
+                          <div>
+                               <h4>Meat</h4>
+                               <p>
+                                   Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                                </p>
+                          </div>
+                      </div>
+                      <div class="cards">
+                           <img src="./components/service3.jpg" alt="service3 image">
+
+                          <div>
+                               <h4>Hamburger</h4>
+                               <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                           </div>
+                      </div> 
+                      <div class="cards">
+                          <img src="./components/bg2.jpg" alt="service4 image">
+                           <div>
+                               <h4>Salade</h4>
+                               <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                           </div>
+                      </div> 
+                    </div>
+                 </aside>
```

Comment:
- [ ] In order to respect the design, kindly, implement another card in the Signature dishes section;

Your design:
![image](https://user-images.githubusercontent.com/83596135/163894026-e02d2132-0af2-4f4f-95d6-24bbee5b7b71.png)

Given design:
![image](https://user-images.githubusercontent.com/83596135/163894070-6c5c403a-6382-4330-a555-1875246b82e5.png)

Reply:
- [ ] Also, in order to respect the design, please add a anchor tag/button under the cards:

Given design for desktop:
![image](https://user-images.githubusercontent.com/83596135/163894284-2e263018-5ea5-470a-80b9-cb12ba86190f.png)

Given design for mobile:
![image](https://user-images.githubusercontent.com/83596135/163894230-6740b8b6-c9f0-47e9-9d3f-81d26efb1102.png)

Reply:
Hello @nicupop729, 
I thank you for your feedback, I did all the changes requested.